Packs/ApiModules/Scripts/CSVFeedApiModule/CSVFeedApiModule.py

```diff
@@ -264,14 +269,14 @@ def fetch_indicators_command(client: Client, default_indicator_type: str, **kwar
                         'fields': create_fields_mapping(raw_json, mapping) if mapping else {}
                     }
                     indicators.append(indicator)
-
     return indicators
 
 
 def get_indicators_command(client, args):
     itype = args.get('indicator_type', demisto.params().get('indicator_type'))
     limit = int(args.get('limit'))
-    indicators_list = fetch_indicators_command(client, itype)
+    auto_detect = args.get('auto_detect_type')
```

Comment:
Why are you adding an argument for `auto_detect_type` for the command (and obviously in the generic `CSV Feed` integration YAML as well)? The command can just account for the `auto_detect_type` parameter of the integration (which you seem to have removed from that YAML... not sure why you did that - need to put it back) and you would need to update the tooltip or description of the `indicator_type` argument of the command to say that if `auto_detect_type` integration parameter is marked as true (checkmark) then the `indicator_type` argument will be ignored.

Reply:
And if you want the `auto_detect_type` parameter to be accessible via the `args` dictionary passed to `get_indicators_command`, then just update the `args` dictionary in `feed_main` to include that key/value as taken from `demisto.params()`.